cmd/olfullnode/devnet.go

```diff
@@ -738,19 +740,24 @@ func initialState(args *testnetConfig, nodeList []node, option ethchain.ChainDri
 	}
 }
 
-func deployethcdcontract(conn string, nodeList []node) (*ethchain.ChainDriverOption, error) {
-
-	f, err := os.Open(os.Getenv("ETHPKPATH"))
-	if err != nil {
-		return nil, errors.Wrap(err, "Error Reading File")
-	}
-	b1 := make([]byte, 64)
-	pk, err := f.Read(b1)
-	if err != nil {
-		return nil, errors.Wrap(err, "Error reading private key")
+func deployethcdcontract(conn string, nodeList []node, ethprivKey string) (*ethchain.ChainDriverOption, error) {
+	pkStr := ""
+	if len(ethprivKey) == 0 {
```

Comment:
I just realised ,in the future if we plan to use this setup for CI as well . It might not be such a good idea to be passing the the private key as a simple string .
We can probably sign trasactions using a hd wallet module to avoid this. 

Does not need to be a part of this PR , but let me know your thoughts on this.